setup.py

```diff
@@ -35,6 +35,16 @@
 )
 from pysetup.md_doc_paths import get_md_doc_paths
 
+# Ignore '1.5.0-alpha.*' to '1.5.0a*' messages.
+import warnings
+warnings.filterwarnings('ignore', message='Normalizing .* to .*')
+
+# Ignore 'running' and 'creating' messages
+import logging
+class PyspecFilter(logging.Filter):
+    def filter(self, record):
+        return not record.getMessage().startswith(('running ', 'creating '))
+logging.getLogger().addFilter(PyspecFilter())
```

Comment:
I feel like this stuff tends to be fragile so it could more easily break in the future

Reply:
Yeah maybe. Shouldn't be difficult to update if something breaks. Worst case we can just delete it.